config/tileset/health.json

```diff
@@ -3,7 +3,7 @@
   "name": "health",
   "background": "dce9edff",
   "layers": [
-    { "2193": "01EDD8XJZ5TPY1SKPC26B1CQNW", "3857": "01EDMTM3P563P06TWYQAZRA9F6", "name": "gebco_2020_305-75m", "maxZoom": 15 },
+    { "2193": "01F1BFJN8R8P7BXN3XTHC5MT5G", "3857": "01EDMTM3P563P06TWYQAZRA9F6", "name": "gebco_2020_305-75m", "maxZoom": 15 },
```

Comment:
Why do we update this? @blacha

Reply:
so we don't have two different 2193 gebco layers imported

Will have to see if this breaks the health endpoint

Reply:
Cool. Thanks.